packages/react-button/etc/react-button.api.md

```diff
@@ -286,7 +286,7 @@ export type ToggleButtonVariants = ButtonVariants<ToggleButtonTokens>;
 export const useButton: (props: ButtonProps, ref: React.Ref<HTMLElement>, defaultProps?: ButtonProps | undefined) => ButtonState;
 
 // @public (undocumented)
-export const useButtonClasses: (state: ButtonState, options?: import("@fluentui/react-theme-provider").UseStylesOptions | undefined) => void;
+export const useButtonClasses: (state: ButtonState, options?: import("@fluentui/react-theme-provider/lib/makeStyles.compat").UseStylesOptions | undefined) => void;
```

Comment:
No `/lib/` paths in converged components.  This is further tying us to build/dev workflow problems. See https://hackmd.io/_NEjeq1vQomhhfKTImYL9g